components/markets/market-card/index.tsx

```diff
@@ -268,14 +268,26 @@ const MarketCard = ({
                 pool={pool}
                 prediction={prediction}
               />
-            ) : (
+            ) : pool !== null ? (
```

Comment:
Can pool be `undefined`?

Reply:
I don't think so. When I run the markets query in the graphiql explorer it comes back with either `null` or with values inside the `pool `object. Do you think it's worth adding a check for `undefined` to be safe?